RecoHGCal/TICL/plugins/TrackstersPCA.cc

```diff
@@ -52,10 +59,22 @@ void ticl::assignPCAtoTracksters(std::vector<Trackster> &tracksters,
       fillPoint(layerClusters[trackster.vertices(i)], weight);
       for (size_t j = 0; j < 3; ++j)
         barycenter[j] += point[j];
+
+      // Also compute timing
```

Comment:
The previous implementation was rather verbose, but this one is Ungaretti's style, but way less expressive.
At the very least, add a comment to explain we skip already used LayerClusters from the timing-computation, to clarify the meaning of line 64.